lib/TraackrApi/TraackrApiObject.php

```diff
@@ -9,6 +9,25 @@
 
    private $curl;
 
+   // Headers passed with each request
+   private $curl_headers  = array(
```

Comment:
@bbguitar77 these are not the headers you are looking for:
![mindtrick-1](https://cloud.githubusercontent.com/assets/1584090/8864174/d19e80f8-3153-11e5-969a-fe0b5df31f87.gif)